packages/js/product-editor/src/blocks/product-fields/name/edit.tsx

```diff
@@ -156,6 +158,36 @@ export function Edit( {
 		}
 	}, [] );
 
+	const [ featured, setFeatured ] =
+		useProductEntityProp< boolean >( 'featured' );
+
+	function handleSuffixClick() {
+		setFeatured( ! featured );
+	}
+
+	function renderFeaturedSuffix() {
+		return (
+			<Tooltip
+				text={ __( 'Mark as featured', 'woocommerce' ) }
```

Comment:
I am ok leaving this as is, but given we do use `Unmark as featured` for the aria label below, I was wondering if we should update this to say `Unmark` when it is selected?

Reply:
Done -> https://github.com/woocommerce/woocommerce/pull/43664/commits/e0b8bc426ec441760c51619ccc79772d714b595c